stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/search/SearchResultListView.kt

```diff
@@ -49,12 +50,12 @@ public class SearchResultListView : ViewFlipper {
         binding.searchListView.apply {
             setHasFixedSize(true)
             adapter = this@SearchResultListView.adapter
-            addItemDecoration(
-                DividerItemDecoration(
-                    context,
-                    LinearLayoutManager.VERTICAL
-                )
-            )
+
+            val configuration = resources.configuration
+            if (configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK != Configuration.UI_MODE_NIGHT_YES) {
+                addItemDecoration(DividerItemDecoration(context, LinearLayoutManager.VERTICAL))
+            }
```

Comment:
Looking at the design, I think dark mode also has dividers, their colour is just different.